README.md

```diff
@@ -8,156 +15,103 @@
-|---|---|------------------|
-| snowflake | pip install 'piperider[snowflake]'  |                  |
-| postgres  | pip install 'piperider[postgres]'  |                  |
-| bigquery | pip install 'piperider[bigquery]'  | PipeRider v0.7.0 |
-| redshift | pip install 'piperider[redshift]'  | PipeRider v0.7.0 |
-| parquet | pip install 'piperider[parquet]' | PipeRider v0.8.0 |
-| csv | pip install 'piperider[csv]' | PipeRider v0.8.0 |
-| duckdb | pip install 'piperider[duckdb]' | PipeRider v0.8.0 |
+| connectors | install                              |
+| ---------- | ------------------------------------ |
+| snowflake  | `pip install 'piperider[snowflake]'` |
+| postgres   | `pip install 'piperider[postgres]'`  |
+| bigquery   | `pip install 'piperider[bigquery]'`  |
+| redshift   | `pip install 'piperider[redshift]'`  |
+| parquet    | `pip install 'piperider[parquet]'`   |
+| csv        | `pip install 'piperider[csv]'`       |
+| duckdb     | `pip install 'piperider[duckdb]'`    |
 
 Use comma to install multiple connectors in one line:
 
 ```bash
 pip install 'piperider[postgres,snowflake]'
 ```
 
-You can follow the [quick start guide](https://docs.piperider.io/quick-start) to learn more about PipeRider.
+## Initialize Project & Diagnose Settings
 
-## Attach PipeRider to a dbt project
+Once installed, initialize a new project with the following command.
 
-`piperider init` creates `/.piperider` under a dbt project root and generates necessary configurations.
-
-## Scan data quality from models
-
-`piperider run` runs profiling, and tests against assertions, and will render an HTML report `/.piperider/outputs`
-
-You can do a run for a specific table using `piperider run --table $TABLENAME`
-You can specifying the output location of a report using `piperider generate-report -o $PATHNAME` or specifying it in the config file.
-
-[All details on generating reports](https://docs.piperider.io/how-to-guides/generate-report)
+```bash
+piperider init        # initializes project config
+piperider diagnose    # verifies your data source connection & project config
+```
 
-## Compare reports
+## Profiling and Testing Your Data
 
-You can use `piperider compare-reports` to compare 2 reports.
+Next, execute `piperider run`, which will do a number of things:
 
-# Learn More
+1. Create a data profile of your data source
+1. Auto-generate recommended or template assertions files (first-run only)
+1. Test that data profile against any available assertions, including custom and/or recommended assertions
+1. Generate a static HTML report, which helps visualize the data profile and assertion results.
 
-| PipeRider Resources | Description |
-| -------------------- | ----------- |
-| [Documentation] | PipeRider Main Doc Site |
-| [Sample_Project] | Sample Project with with sqlite |
-| [dbt_Sample_Project] | Sample Project with dbt |
-| [Roadmap] | PipeRider Roadmap |
-| [Blog] | How we got started |
+Common Usages/Tips:
```

Comment:
I like the new structure, it's simpler and more focused on what we want to provide.

Suggestion:
Both Common Usages/Tips parts could use code blocks instead of using list items.
It will be more consistent due to we already put code blocks in the Installation and Initialize Project session.